package.json

```diff
@@ -176,7 +176,7 @@
     "tar": "^6.1.11",
     "ts-jest": "27.1.5",
     "ts-node": "~10.9.1",
-    "typescript": "4.9.5",
+    "typescript": "5.6.2",
```

Comment:
Should we consider removing it from here? This is a guard that forces using the same TS version across monorepo.

Reply:
I think we need it here because without it, the top-level `tsconfig.json` won't know what configuration options are avaialable. Is that right @SokratisVidros ?

Reply:
@rifont I tried that in the other PR and it was fine 🤷‍♂️

Reply:
@rifont you are correct. If we remove it from here VS code gets a bit confused. The other thing I tried is to use `typescript: "*"` and then just set a fixed TS at the top level but I read that it's not advisable for the publishable packages as they end up in NPM. TBH I don't get it as it's a dev dependency but I didn't delve deeper.